docs/samples/contact-center/app.js

```
@@ -557,59 +559,134 @@ function registerTaskListeners(task) {
+  const hasParticipants = Object.keys(participants).length > 1;
+  endElm.disabled = !hasParticipants;
+  if (task.data.interaction.mediaType === 'chat' || task.data.interaction.mediaType === 'email') {
+    holdResumeElm.disabled = true;
+    muteElm.disabled = true;
+    pauseResumeRecordingElm.disabled = true;
+    consultTabBtn.disabled = true;
+    declineElm.disabled = true;
+    transferElm.disabled = false;
+  } else if (currentTask.data.interaction.mediaType == 'telephony') {
+    // hold/resume call
+    const isHold = media && media[mediaResourceId] && media[mediaResourceId].isHold;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)